src/node_url.cc

```diff
@@ -1173,30 +1181,32 @@ bool ParseHost(const std::string& input,
   return true;
 }
 
-inline std::vector<std::string> FromJSStringArray(Environment* env,
-                                                  Local<Array> array) {
-  std::vector<std::string> vec;
-  const int32_t len = array->Length();
-  if (len == 0)
-    return vec;  // nothing to copy
-  vec.reserve(len);
-  for (int32_t n = 0; n < len; n++) {
+std::vector<std::string> FromJSStringArray(Environment* env,
+                                           Local<Array> array) {
+  std::vector<std::string> vec(array->Length());
```

Comment:
Fwiw, the previous version avoided allocating `len` × empty `std::string`, unlike this one… I would be good with keeping it?

Reply:
Ok, restored the original behavior!